store/modules/forecastViz.js

```diff
@@ -60,14 +77,26 @@ export default moduleOptions => ({
       }
       this.dispatch('forecastViz/fetch_as_of_truth');
       this.dispatch('forecastViz/fetch_forecasts');
```

Comment:
same comment here about adding await on these dispatches and declaring the function as `async`
```suggestion
      await this.dispatch('forecastViz/fetch_as_of_truth');
      await this.dispatch('forecastViz/fetch_forecasts');
```

Reply:
I added ESLint here because this is the most recent code and I wanted to avoid merge conflicts. What I understand is that we should add an await for all the dispatch calls right?
I was not sure what to do with the previous code, I'll delete that.